dftimewolf/lib/exporters/scp_ex.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""Collect artifacts from the local filesystem."""
+
+from __future__ import unicode_literals
+
+import subprocess
+
+from dftimewolf.lib.module import BaseModule
+
+class Scp(BaseModule):
+  """Copies the files in the previous module's output to a given path.
```

Comment:
Please make sure the docstring follows l2t's styleguide: https://github.com/log2timeline/l2tdocs/blob/master/process/Style-guide.md#class-attributes

(looks like you forgot the `Attributes:` section)